### PR TITLE
fix pagination

### PR DESCRIPTION
**Pull request by sanglevinh**

### CODE REVIEW

File: frontend/src/components/Database/DatabaseCells.tsx [hunk 1]
```diff
@@ -278,9 +278,7 @@ const DatabaseCells = ({ user }: CellProps) => {
   const handleLimit = (event: ChangeEvent<HTMLSelectElement>) => {
     let filter = ''
     filter = Object.keys(dataParamsFilter).filter(key => (dataParamsFilter as any)[key])
-        .map((item: any) => {
-          return `${item.field}=${item?.value}`
-        })
+        .map((item: any) => `${item}=${(dataParamsFilter as any)[item]}`)
```
Comment (itutu-tienday) [inline frontend/src/components/Database/DatabaseCells.tsx:281]: @sanglevinh 

Which issue is this correcting?
#143, right? However, it seems that the issue has not yet been resolved as follows.

image

**Review by itutu-tienday — CHANGES_REQUESTED** [review]

File: frontend/src/components/Database/DatabaseCells.tsx (hunk 1 above)
Comment (sanglevinh) [reply frontend/src/components/Database/DatabaseCells.tsx:281]: @itutu-tienday this branch i created to fix the error when changing limit the number of records is showing wrong, params problem i fixed in PR https://github.com/arayabrain/optinist-for-server/pull/160

Comment (itutu-tienday) [reply frontend/src/components/Database/DatabaseCells.tsx:281]: @sanglevinh 
Let me check the status of this PR.
-> Is it assumed to be open or closed?

Comment (sanglevinh) [reply frontend/src/components/Database/DatabaseCells.tsx:283]: @itutu-tienday I have changed the open status to PR, please check it again

**Review by ReiHashimoto — APPROVED** [review]
LGTM

**Review by itutu-tienday — APPROVED** [review]

**Pull request merged**